### PR TITLE
Upgrade dependencies

### PR DESCRIPTION
**Pull request by hug-dev**
Signed-off-by: Hugues de Valon <hugues.devalon@arm.com>

### CODE REVIEW

**Comment by hug-dev** [comment]
There is a bug in `psa-crypto` when it is compiled with `--no-default-features --features interface`, it does not compile.

**Comment by hug-dev** [comment]
This will need to pull the newer version of `psa-crypto` once parallaxsecond/rust-psa-crypto#61 is merged.

**Comment by hug-dev** [comment]
CI fails because of parallaxsecond/rust-psa-crypto#63

File: .travis.yml [hunk 1]
```diff
@@ -1,5 +1,6 @@
 # Executing our tests on Arm64 with Travis CI
 arch: arm64
+dist: bionic
```
Comment (hug-dev) [inline .travis.yml:3]: Was needed to have a recent version of `libclang`

**Review by ionut-arm — APPROVED** [review]
LGTM!

**Comment by hug-dev** [comment]
Added a `patch` section in the `Cargo.toml`, to be replaced with a newer version of `psa-crypto` when released.

**Pull request merged**